lib/gui-utils/src/GUIUtils.cpp

```diff
@@ -93,103 +118,292 @@ void GUIUtils::displayEndLine(String msg) {
   u8g2.print(msg.c_str());
 }
 
-void GUIUtils::displaySensorAvarage(int avarage) {
+void GUIUtils::displaySensorAvarage(int avarage)
+{
+#ifndef Emoticons
+#ifdef TTGO_TQ
+  if (avarage < 13)
+  {
+    u8g2.setFont(u8g2_font_unifont_t_emoticons);
+    u8g2.drawGlyph(76, 12, 0x0024);
+    u8g2.setFont(u8g2_font_5x7_tf);
+    u8g2.setCursor(77, 16);
+    u8g2.print("GOOD");
+  }
+  else if (avarage < 36)
+  {
+    u8g2.setFont(u8g2_font_unifont_t_emoticons);
+    u8g2.drawGlyph(76, 12, 0x0062);
+    u8g2.setFont(u8g2_font_4x6_tf);
+    u8g2.setCursor(77, 17);
+    u8g2.print("MODERATE");
+  }
```

Comment:
please specialize it in two methods for not have repeated code, for example something like that:
```suggestion
  {
    displayEmoticonLabel(0x0032,"UNHEALT S")
  }
  else if (avarage < 36)
  {
    displayEmoticonLabel(76,12,0x0062,"MODERATE");
  }
```

Reply:
Resolved:
https://github.com/danielbernalb/esp32-hpma115s0/commit/7c5910c159f0f4f362b51306db3957029769818b